examples/examples_yaml_test.go

```diff
@@ -49,6 +50,7 @@ func TestWarningsNotDuplicated(t *testing.T) {
 		SkipRefresh:   true,
 		Env:           []string{"GOOGLE_PROJECT=", "GOOGLE_APPLICATION_CREDENTIALS="},
 		ExtraRuntimeValidation: func(t *testing.T, stackInfo integration.RuntimeValidationStackInfo) {
+			fmt.Printf("output: %v\n", outputBuf.String())
```

Comment:
Nit: Use testing's built-in logging.
```suggestion
			t.Logf("output: %v\n", outputBuf.String())
```